migrations/007-create-article.js

```diff
@@ -0,0 +1,38 @@
+const tableName = 'Articles';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => Promise.all([
+    queryInterface.removeColumn(tableName, 'post'),
+
+    queryInterface.addColumn(tableName, 'title', {
+      type: Sequelize.STRING,
+      allowNull: false
+    }),
+
+    queryInterface.addColumn(tableName, 'body', {
+      type: Sequelize.STRING,
+      allowNull: true
+    }),
+
+    queryInterface.addColumn(tableName, 'description', {
+      type: Sequelize.STRING,
+      allowNull: true
+    }),
+
+    queryInterface.addColumn(tableName, 'tags', {
+      type: Sequelize.STRING,
+      allowNull: true
+    }),
+
+    queryInterface.addColumn(tableName, 'slug', {
+      type: Sequelize.STRING,
+      allowNull: true
+    })
+  ]),
+
+  /* eslint-disable no-unused-vars */
+  down: (queryInterface, Sequelize) => Promise.all([
+    queryInterface.removeColumn(tableName, 'slug'),
```

Comment:
All newly added columns should be removed in this section, while all removed columns should be added.

Reply:
I've made the changes. I didn't think that if the columns to be removed were added in this section, they'd be run automatically, alongside `db:migrate`